include/dynamatic/Dialect/Handshake/HandshakeOps.td

```diff
@@ -1093,6 +1110,7 @@ def BundleOp : Handshake_Op<"bundle", [Pure]> {
 
   let skipDefaultBuilders = 1;
   let builders = [
+    // A builder for bundling ControlType
```

Comment:
using a full sentence "a builder for bundling ControlType" makes this feel like a descriptive comment, but I think you are making a label comment here, which I would change to:
"ControlType Builder"

the information isn't any different, but I think people interpret it differently when they read it

Reply:
ControlType Builder sounds like `ControlType::get` to me.
Do you mean ControlType Bundler? I like this

Reply:
Oh yeah I misread. I think most people will misread the same way I did, so the comment should be short, have no a's or the's, and highlight that it is for bundling as much as possible